docs/Subcommand-System.md

```diff
@@ -0,0 +1,136 @@
+# Homebrew Subcommand System
```

Comment:
Don't want this as public documentation.

Reply:
removed